src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -443,11 +443,11 @@ protected String getDisplaySummaryForTool(final Class<?> toolClass, final Comman
         final String summaryLine;
 
         if (experimentalFeature != null) {
-            summaryLine = String.format("%s%s %s%s", RED, "(EXPERIMENTAL Tool)", CYAN, clpProperties.oneLineSummary());
+            summaryLine = String.format("%s%s %s%.120s", RED, "(EXPERIMENTAL Tool)", CYAN, clpProperties.oneLineSummary());
```

Comment:
Do we really want to truncate the descriptions rather than wrap them? I'm not convinced this is better...

Reply:
+1 to @droazen suggestion of wrapping and/or do not bound and just add a test. Otherwise, this wrapping should be configurable somehow for downstream toolkits...